logstash-core/lib/logstash/agent.rb

```diff
@@ -110,7 +110,7 @@ def pipeline_batch_delay=(pipeline_batch_delay_value)
 
   def debug_config=(debug_config)
     @config_loader.debug_config = debug_config
-    @debug_config
+    @debug_config = true
```

Comment:
any reason to do `@debug_config = true` instead of `@debug_config = debug_config`?